README.md

```diff
@@ -10,3 +9,33 @@ For detailed API description, please see [the draft on the project
 wiki](https://github.com/codingteam/Gestalt/wiki/Public-API).
 
 For copying terms, see LICENSE file.
+
+# Buildig and running the service
+
+You need the following installed in your system:
+
+* [Erlang version 17 or above](erlang.org)
+* [rebar](https://github.com/rebar/rebar/wiki/Getting-started#first-steps)
+
+Now run:
+
+    # compile the source and generate the release
+    $ rebar compile generate
+
+    # start the service
+    $ ./rel/gestalt/bin/gestalt start
```

Comment:
SO [suggests](https://stackoverflow.com/questions/17442232/erlang-project-cant-launch-on-windows-generated-by-rebar) the following workaround:

    $ ./rel/gestalt/bin/gestalt install
    $ ./rel/gestalt/bin/gestalt start

or, if you don't want to install it:

    $ ./rel/gestalt/bin/gestalt console

Please check if this works so that I can update the README.